stacks-core/release/check-release/action.yml

```
@@ -0,0 +1,80 @@
+## Composite action to check if the branch is a release tag or not
+name: Check Release
+
+inputs:
+  tag:
+    description: "The branch name against which the step runs"
+    required: false
```

```suggestion
    required: true
```
looking over the shell below, i think this should be required